addons/test_discuss_full/tests/test_performance.py

```diff
@@ -429,6 +435,7 @@ def _get_init_messaging_result(self):
                                         'user': {
                                             'id': self.users[12].id,
                                             'isInternalUser': True,
+                                            'notificationType': 'inbox'
                                         },
```

Comment:
User 0 has `notificationType: 'inbox'` but the rest like user 12 has default, which is `'email'`.